packages/boot/tools/ibc/mocks.js

```diff
@@ -7,22 +52,85 @@
  */
 export const icaMocks = {
   startChannelOpenInit: {
-    // ICA Channel Creation
-    channelOpenAck: obj => ({
-      type: 'IBC_EVENT',
-      blockHeight: 99,
-      blockTime: 1711571357,
-      event: 'channelOpenAck',
-      portID: obj.packet.source_port,
-      channelID: 'channel-0',
-      counterparty: {
-        port_id: obj.packet.destination_port,
-        channel_id: 'channel-1',
-      },
-      counterpartyVersion:
-        '{"version":"ics27-1","controllerConnectionId":"connection-0","hostConnectionId":"connection-0","address":"osmo1234","encoding":"proto3","txType":"sdk_multi_msg"}',
-      connectionHops: obj.hops,
-    }),
+    /**
+     * ICA Channel Creation
+     * @param {ChannelOpenInitPacket} obj
+     */
+    channelOpenAck: obj => {
+      const icaControllerNonce = obj?.packet?.source_port?.split('-')?.[1];
```

Comment:
This seems suspicious, since channelId's are frequently different between the subject chain and the counterparty, and also have no relation to the port number.  Could you maybe separate and document what these numbers actually are:
```suggestion
      const sourcePortNum = obj?.packet?.source_port?.split('-')?.at(-1);
      const destPortNum = obj?.packet?.destination_port?.split('-')?.at(-1);
```

Reply:
Agree, I will add your suggestions so this mocking is documented clearly.